package.json

```diff
@@ -9,16 +9,21 @@
         "url": "https://github.com/mozilla/eslint-plugin-no-unsanitized/issues"
     },
     "devDependencies": {
-        "@babel/core": "^7.15.8",
+        "@babel/core": "^7.20.12",
         "@babel/eslint-parser": "^7.15.8",
         "@babel/plugin-proposal-logical-assignment-operators": "^7.11.0",
         "@babel/plugin-proposal-nullish-coalescing-operator": "^7.10.4",
         "@babel/plugin-syntax-flow": "^7.16.0",
+        "@babel/preset-env": "^7.20.2",
         "@typescript-eslint/parser": "^5.2.0",
+        "babel-loader": "^9.1.2",
         "eslint": "^8.1.0",
         "mocha": "^9.2.0",
+        "node-polyfill-webpack-plugin": "^2.0.1",
         "nyc": "^15.1.0",
-        "typescript": "^3.9.7"
+        "typescript": "^3.9.7",
+        "webpack": "^5.75.0",
```

Comment:
**Issue**: this webpack version does have a security advisory associated to it, I haven't checked it by the most recent version isn't affected and so we should just update the version here.

Nit: and while we are here let's also make sure webpack-cli version is the last version released too.